src/lib/ng-package/entry-point/analyse-sources.transform.ts

```diff
@@ -74,7 +74,7 @@ async function analyseEntryPoint(
       }
 
       const moduleName = moduleSpecifier.text;
-      if (moduleName.startsWith(packageNode.data.primary.moduleId)) {
+      if (moduleName.startsWith(`${packageNode.data.primary.moduleId}/`)) {
```

Comment:
Should we test for `moduleName === packageNode.data.primary.moduleId` as suggested by @CarstenLeue?

Reply:
Sry just saw this, yeah definitely. I forgot that part.